packages/flutter_tools/test/src/common.dart

```diff
@@ -145,6 +145,22 @@ Future<void> expectToolExitLater(Future<dynamic> future, Matcher messageMatcher)
   }
 }
 
+Matcher containsIgnoreWhitespace(String toSearch){
+return  predicate((String source)  {
```

Comment:
I think there's an extra space between `return` and `predicate` here?

Reply:
done!